{{cookiecutter.app_name}}/README.rst

```diff
@@ -135,3 +143,52 @@ should cache all your assets forever by including the following line
 in your ``settings.py``::
 
     SEND_FILE_MAX_AGE_DEFAULT = 31556926  # one year
+
+{%- if cookiecutter.use_heroku == "yes" %}
+
+Heroku
+------
+
+Before deployment on the Heroku you should be familiar with a basic concepts of `Git <https://git-scm.com/>`_ and `Heroku <https://heroku.com/>`_.
+
+Remember to add migrations to your repository. Please check `Migrations`_ section.
+
+Since filesystem on Heroku is ephemeral (sqlite database will be lost at least once every 24 hours) the PostgreSQL is a recommended choice. Due to this fact the additional library is required: ``psycopg2-binary``.
+
+**WARNING!** ``psycopg2-binary`` package is a practical choice for development and testing but in production it is advised to use the package built from sources. Read more in the `psycopg2 documentation <http://initd.org/psycopg/docs/install.html?highlight=production%20advised%20use%20package%20built%20from%20sources#binary-install-from-pypi>`_
+
+Deployment by using `Heroku CLI <https://devcenter.heroku.com/articles/heroku-cli>`_:
+
+* create Heroku App. You can leave your app name, change it or leave it blank (random name will be generated)::
+
+    heroku create {{cookiecutter.app_name}}
+
+* add buildpacks::
+
+    heroku buildpacks:add --index=1 heroku/nodejs
+    heroku buildpacks:add --index=1 heroku/python
+
+* add database addon which sets PostgresSQL in version 11 in free `hobby-dev <https://elements.heroku.com/addons/heroku-postgresql#hobby-dev>`_ plan (it also sets ``DATABASE_URL`` environmental variable to created database)::
+
+    heroku addons:create heroku-postgresql:hobby-dev --version=11
+
+* set environmental variables (change ``SECRET_KEY`` value for your own)::
+
+    heroku config:set SECRET_KEY=not-so-secret
+    heroku config:set FLASK_APP=autoapp.py
+
+*   ``DATABASE_URL`` environmental variable is set by database addon.
+    Please check ``.env.example`` to get overview which environmental variables are used in the project.
+
+* deploy on Heroku by pushing to the ``heroku`` branch::
+
+    git push heroku master
+
+If you keep your project on GitHub you can use 'Deploy to Heroku' button thanks to which the deployment can be done in web browser with minimal configuration required.
```

Comment:
How about switching the order of this button with the CLI instructions, so that it comes first in the section?

Reply:
Sure, good idea! I've changed order, added `style="display: block"` to anchor and added `<br>` to get some additional space between button and the following instructions, but I am not sure did it help too much.